### ISSUE TITLE
Any way to update this?

### ISSUE THREAD

**Issue by camdenorrb** [Kozova1/parsec-solus#1]
Is there any manual way to update this package? Is there any need to worry about it becoming outdated?

**Comment by hundsboog** [comment]
Yes, an update would be very cool. It seems, that something is not working. When I enter my credentials, the terminal closed itself.
